app/src/components/idpService.js

```diff
@@ -0,0 +1,169 @@
+const errorToProblem = require('./errorToProblem');
```

Comment:
Can we get unit tests added for idpService and jwtService? I think that Code Climate is going to complain about code complexity, and having the tests will make future refactoring easier.

Reply:
seems like a great idea. will do.

Reply:
Looks great, thank you!